### DISCUSSION TITLE
Metadata resolution display in portrait video

### DISCUSSION THREAD

**Discussion by chamdim** [immich-app/immich#14920] [General]
Even if a video is recorded in portrait mode, its resolution is displayed in its metadata as 1920x1080, for example, when it should be shown as 1080x1920. This issue does not occur with photos, where the resolution is displayed correctly.

**Comment by YarosMallorca** [comment]
I can confirm this is the case, only on mobile though.

EDIT:
This issue also applies to images, not only videos. The resolution format is swapped on mobile.
I made a PR fixing this issue: https://github.com/immich-app/immich/pull/14954
